src/components/ItemDetailPage/ItemDetailPage.tsx

```diff
@@ -51,6 +59,12 @@ export default class ItemDetailPage extends React.PureComponent<Props> {
                       direction="left"
                     >
                       <Dropdown.Menu>
+                        {missingBodyShape !== null ? (
+                          <Dropdown.Item
+                            text={t('item_detail_page.add_representation', { bodyShape: t(`global.representation.${missingBodyShape}`) })}
```

Comment:
I know it wasn't introduced in this PR but maybe this should be called `
```
t(`bodyShapes.${missingBodyShape}`)
```
on the translations file?

Reply:
done 👍